sensor_msgs/Image image
---
geometry_msgs/Twist motor_control